src/fragments/start/getting-started/reactnative/integrate.mdx

```diff
@@ -0,0 +1,411 @@
+In this section you’ll integrate Amplify DataStore with your app, and learn to use the generated data model to create, update, query, and delete Todo items by building an app. You can find the source code of the final Todo App [on our GitHub repository](https://github.com/chintannp/amplified_todo_rn).
+
+## Boilerplate UI
+First, we will create some files and place the boilerplate UI code in it. You might find your IDE complains about unreferenced declarations but, don’t worry, we’ll get around to fixing those as we fill out our app with more functionality.
+
+#### App.js
+First, replace the contents of your `App.js` file with the following UI boilerplate code. Here, we are importing our Home component. Additionally, we are configuring Amplify using `Amplify.configure`. You might have already done this in previous step while setting up the project.
+
+```js
+import React from 'react';
+import { StatusBar, StyleSheet, View } from 'react-native';
+import Amplify from 'aws-amplify';
+import Home from './src/Home';
+
+import awsconfig from './src/aws-exports';
+
+Amplify.configure(awsconfig);
```

Comment:
Would it actually be best practice to configure this in `index.js` instead?

Reply:
It is best practice for React Native CLI. I have updated the guide to use index.js for RN CLI. However, Expo does not have index.js file. So, App.js will be used for Expo.